DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -145,11 +139,30 @@ DlsSubApplication dlsSubApplication
                 return ReturnToEditDetailsViewWithErrors(formData, dlsSubApplication);
             }
 
-            if (!userService.NewEmailAddressIsValid(formData.Email!, userAdminId, userDelegateId, User.GetCentreId()))
+            if (!userService.NewEmailAddressIsValid(
+                    formData.Email!,
+                    userAdminId,
+                    userDelegateId,
+                    userId
+                ))
             {
                 ModelState.AddModelError(
                     nameof(MyAccountEditDetailsFormData.Email),
-                    "A user with this email address is already registered at this centre"
+                    "This email address is already in use"

```

Comment:
Maybe we could add this error message to the CommonValidationErrorMessages helper where the other email-related error messages are?